google-cloud-dialogflow/src/test/java/com/google/cloud/dialogflow/v2/it/ITSystemTest.java

```diff
@@ -0,0 +1,323 @@
+  private static final String ENTITY_NAME = "test-entity-" + ID;
+  private static final EntityType.Entity ENTITY =
+      EntityType.Entity.newBuilder().addSynonyms("test-entity").setValue("test-entity").build();
+  private static final String INTENT_NAME = "test-intent-" + ID;
+  private static final String EVENT_NAME = "test-event";
+  private static final String ACTION_NAME = "test-action";
+  private static final String SESSION_ID = UUID.randomUUID().toString();
+  private static final SessionName SESSION_NAME = SessionName.of(PROJECT_ID, SESSION_ID);
+  private static final String CONTEXT_ID = "test-context-" + ID;
+  private static final ContextName CONTEXT_NAME =
+      ContextName.of(PROJECT_ID, SESSION_ID, CONTEXT_ID);
+  private static final Integer LIFE_SPAN_COUNT = 10;
+
+  @BeforeClass
+  public static void beforeClassTest() throws IOException {
+
+    /* create agent */
+    agentsClient = AgentsClient.create();
+    Agent agent =
+        Agent.newBuilder()
+            .setParent(PROJECT_NAME.toString())
+            .setDisplayName(DISPLAY_NAME)
+            .setDefaultLanguageCode(DEFAULT_LANGUAGE_CODE)
+            .setTimeZone(TIME_ZONE)
+            .setMatchMode(Agent.MatchMode.MATCH_MODE_HYBRID)
+            .setApiVersion(Agent.ApiVersion.API_VERSION_V2)
+            .setTier(Agent.Tier.TIER_STANDARD)
+            .build();
+    SetAgentRequest agentRequest = SetAgentRequest.newBuilder().setAgent(agent).build();
+    agentsClient.setAgent(agentRequest);
+
+    /* create entity */
+    entityTypesClient = EntityTypesClient.create();
+    EntityType createEntityType =
+        EntityType.newBuilder()
+            .setDisplayName(ENTITY_NAME)
+            .setKind(EntityType.Kind.KIND_LIST)
+            .addEntities(ENTITY)
+            .build();
+    CreateEntityTypeRequest entityTypeRequest =
+        CreateEntityTypeRequest.newBuilder()
+            .setParent(PROJECT_AGENT_NAME.toString())
+            .setEntityType(createEntityType)
+            .build();
+    entityType = entityTypesClient.createEntityType(entityTypeRequest);
+    entityName =
+        entityType.getName().substring(entityType.getName().lastIndexOf("/")).replace("/", "");
+    entityTypeName = EntityTypeName.of(PROJECT_ID, entityName);
+
+    /* create intents */
+    intentsClient = IntentsClient.create();
+    Intent createIntent =
+        Intent.newBuilder()
+            .setDisplayName(INTENT_NAME)
+            .addEvents(EVENT_NAME)
+            .setAction(ACTION_NAME)
+            .build();
+    CreateIntentRequest intentRequest =
+        CreateIntentRequest.newBuilder()
+            .setParent(PROJECT_AGENT_NAME.toString())
+            .setIntent(createIntent)
+            .build();
+    intent = intentsClient.createIntent(intentRequest);
+    intentName = intent.getName().substring(intent.getName().lastIndexOf("/")).replace("/", "");
+    intent_name = IntentName.of(PROJECT_ID, intentName);
```

Comment:
whats the diff b/w intentName vs intent_name?

Reply:
Done.